developer/src/tike/child/Keyman.Developer.UI.UframeWordlistEditor.pas

```diff
@@ -217,6 +223,48 @@ procedure TframeWordlistEditor.EnableControls;
   cmdSortByFrequency.Enabled := e;
 end;
 
+function TframeWordlistEditor.ExecuteAction(Action: TBasicAction): Boolean;
+begin
+  if Assigned(modActionsMain) and (Action = modActionsMain.actEditPaste) then
+  begin
+    Result := PasteClipboardIntoGrid;
+  end
+  else
+    Result := False;
+end;
+
+function TframeWordlistEditor.PasteClipboardIntoGrid: Boolean;
+var
+  Value: string;
+begin
+  if FSetup > 0 then
+    Exit(False);
+
+  Inc(FSetup);
+  try
+    try
+      Value := Clipboard.AsText;
+    except
+      on E:EClipboardException do
+        Exit(False);
+    end;
+
+    if not Value.Contains(#9) and not Value.Contains(#13) then
```

Comment:
I'm assuming magic numbers for tab and carriage return, right?
Should we also handle line feed?

Reply:
I guess we could handle #10 as well but it's not really ever used without #13 in Windows.